tests/testsuite.py

```diff
@@ -73,6 +73,10 @@ def as_map(children_values, key_func=to_python_value, value_func=to_python_value
     it = iter(children_values)
     return [(key_func(key), value_func(value)) for ((key_text, key), (value_text, value)) in zip(it, it)]
 
+def as_multi_index(children_values):
```

Comment:
I believe that a separate `as_multi_index()` function is not needed and `as_array(children, int)` should be used in testcases.